helm/cluster/values.schema.json

```diff
@@ -2567,6 +2567,12 @@
                         },
                         "preKubeadmCommands": {
                             "$ref": "#/$defs/preKubeadmCommands"
+                        },
+                        "users": {
```

Comment:
nitpick: I would probably expect that under `users` I would have to set a list of users.

And here for the flag approach maybe call this var `enableGiantswarmUser` or something like that.

Naming is hard 🙈 

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExY3ZhenI3MTQ5YWoyeXcyeGV4NWw2MDAxcDJ2MDg0Yzk0OWRjZ3N6ZSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/BoDVjTJiwICJ8RYbov/giphy.gif)

Reply:
yeah I wasn't sure about the naming either tbh, I ended up with `users` because it aligns with that key in the ignition config. Changing it to `enableGiantswarmUser` would definitely make its usage more obvious.